### PR TITLE
STCOR-873 Ensure support for the passed `tenantId` value by `useChunkedCQLFetch` for manipulations in the context of a specific tenant

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose
https://folio-org.atlassian.net/browse/STCOR-873

useChunkedCQLFetch uses the system tenant under the hood to send requests, however, some places in the application (specifically the consortium manager) require sending requests to a tenant other than the system one. Therefore, useChunkedCQLFetch should be updated to support an additional tenantId option, which will be used by the HTTP client. In the absence of this property, the system tenant will be used by default.

**Note:** PR for `master` branch - https://github.com/folio-org/stripes-core/pull/1518

### CODE REVIEW

File: src/queries/useChunkedCQLFetch.test.js [hunk 1]
```diff
@@ -194,4 +199,37 @@ describe('Given useChunkedCQLFetch', () => {
       expect(result.current.itemQueries?.length).toEqual(2);
     });
   });
+
+  describe('allows work in provided tenant', () => {
+    const providedTenantId = 'providedTenantId';
+
+    it('sets up 1 fetch using alternate ID name', async () => {
```
Comment (zburke) [inline src/queries/useChunkedCQLFetch.test.js:206]: More copy paste problems: you're not changing the ID name here, you're changing the tenantId. How about "sets up 1 fetch using alternate tenant ID"?

**Review by zburke — APPROVED** [review]
LGTM once the comment and test description are corrected to reflect the new `tenantId` argument instead of the copy-pasted code.

**Review by artem-blazhko — APPROVED** [review]

**Comment by zburke** [comment]
@usavkov-epam , it is also worth noting that we have generally managed this branch by regularly rebasing it onto master, which would make this PR unnecessary. If we really really need this ASAP, it's fine to have open a PR as you have done here, but given this work isn't specific to Eureka, the preferred option would be to just pick up this change from master the next time we rebase.

**Comment by usavkov-epam** [comment]
> @usavkov-epam , it is also worth noting that we have generally managed this branch by regularly rebasing it onto master, which would make this PR unnecessary. If we really really need this ASAP, it's fine to have open a PR as you have done here, but given this work isn't specific to Eureka, the preferred option would be to just pick up this change from master the next time we rebase.

Thank you, Zak, for noticing! I wasn't sure how often this branch is updated for changes from `master`, so I decided to create two PRs because these changes are indeed needed for the current feature implementation in the "Keycloak" environment.

**Pull request merged**